ts/metrics/metric.py

```diff
@@ -50,6 +55,8 @@ def __init__(self, name, value,
         self.value = value
         self.dimensions = dimensions
         self.request_id = request_id
+        self.metric_type = metric_type
+        self.is_updated = False if value == 0 else True
```

Comment:
So the main purpose of the `is_updated ` attribute is to determine whether or not the metric gets emitted to the frontend (if the `is_updated` is `True`, then the Metric will be emitted, else it will not). I can understand that `None` may make more sense than using `0`, but my other thought is that a Metric with a value of `0` doesn't provide a lot of information to the user anyhow, so that is why I'm currently using `0`. But if users do want a Metric to have a value of `0`, there is the ability to update the Metric using the `update` method